app/sidebar.py

```diff
@@ -7,7 +7,24 @@
 from ptxboa.api import PtxboaAPI
 
 
+@st.cache_resource()
+def sidebar_logo():
+    st.image("img/agora-energiewende_logo_612x306.png")
+
+
 def make_sidebar(api: PtxboaAPI):
+    st.logo(
+        image="img/logos_wide_text_1000x100.png",  # gets scaled to 240x24 by streamlit
+        icon_image="img/agora-energiewende_logo_612x306.png",
+        link="https://agora-energiewende.org",
+    )
+
+    # st.logo(  # noqa E800
+    #     image="img/logos_wide_no_text_1000x100.png",  # noqa E800
+    #     icon_image="img/logo_agora_no_text_257x100.png",  # noqa E800
+    #     link="https://agora-energiewende.org",  # noqa E800
+    # )  # noqa E800

```

Comment:
I tried the new st.logo. You can uncomment this to get logos with texts removed. The logos don't get larger with st.logo. A larger logo was there with f48b477fd340928d11869ca8149cd880d0e294a1

Reply:
That is too small imo. I would prefer to use just one of the four logos, but with text. Lets have Agora decide which logo they want. And if it still is too small, we can just include an image without using st.logo, right?